jdat_notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_PointSourceDetectorBasedExtraction.ipynb

```diff
@@ -0,0 +1,2258 @@
+{
```

Comment:
<p>If you follow the links to <code>create_distortionMaps.py</code> &amp; <code>PointSourceDetectorBasedExtractionFuncs.py</code> at the top of this cell, you should see that they lead to actual webpages, not a prompt to download the files. This is problematic because the downloaded files are HTML documents, not Python scripts. I can't see their parent folder, but is there a way to use the "download file" link for these instead of "view file?"</p><p> </p><p>The other solution is to copy the scripts into this notebook's setup section in order to avoid downloading these files. Needing to import methods from a script is not common practice in these notebooks, anyway.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
added to repo with git add

Reply:
This was not completely resolved; I moved these imports to the top of the notebook.